include/bbp/sonata/report_reader.h

```diff
@@ -126,8 +129,9 @@ class SONATA_API ReportReader
     mutable std::map<std::string, Population> populations_;
 };
 
+using ElementID = uint32_t;
```

Comment:
shouldn't this be part of common.h ?

Reply:
So you did not move this to common.h at the end ?
What's the reason behind that ?